packages/core/src/shared/featureConfig.ts

```diff
@@ -195,12 +198,22 @@ export class FeatureConfigProvider {
         return this.getFeatureValueForKey(Features.customizationArnOverride).stringValue
     }
 
+    getCustomizationArnOverrideName(): string | undefined {
+        return this.getFeatureVariationForKey(Features.customizationArnOverride)
+    }
+
     // Get the feature value for the given key.
     // In case of a misconfiguration, it will return a default feature value of Boolean true.
     private getFeatureValueForKey(name: FeatureName): FeatureValue {
         return this.featureConfigs.get(name)?.value ?? featureDefinitions.get(name)?.value ?? { boolValue: true }
     }
 
+    // Get the feature variation for the given key.
+    // In case of a misconfiguration, it will return an empty string
+    private getFeatureVariationForKey(name: FeatureName): string {
+        return this.featureConfigs.get(name)?.variation ?? featureDefinitions.get(name)?.variation ?? ''
+    }
+
```

Comment:
can we add a generic function here instead?
``` 
getFeature(feat: string): FeatureContext {
    return this.featureConfigs.get(name)
}
```

Reply:
Updated to call existing static method